packages/typescript/CHANGELOG.md

```diff
@@ -6,22 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Added
+- Add more setup instructions to `@metamask/typescript-eslint-config` README ([#209](https://github.com/MetaMask/eslint-config/pull/209))
+- **BREAKING:** Add rules that require type information ([#250](https://github.com/MetaMask/eslint-config/pull/250))
+  - This requires setting some parser options. See the README for more details.
+- Bump all ESLint dependencies to the latest version ([#252](https://github.com/MetaMask/eslint-config/pull/252))
```

Comment:
Worth clarifying here that we're talking about `peerDependencies` as well, meaning that this change is also breaking